DRAFT.md

```diff
@@ -665,8 +665,7 @@ explicit constexpr indirect()
 
 2. _Mandates_: `T` is a complete type.
 
-3. _Effects_: Constructs an `indirect` owning a default constructed `T`
-  and stores the address in `p`. `alloc` is default constructed.
+3. _Effects_: Equivalent to `indirect(allocator_arg_t{}, Allocator())`.
```

Comment:
`std::vector` has a constructor taking an allocator (but without the `std::allocator_arg_t` parameter: https://eel.is/c++draft/vector#cons

```
constexpr explicit vector(const Allocator&) noexcept;
[1](https://eel.is/c++draft/vector#cons-1)
[#](http://github.com/Eelis/draft/tree/7faaaf22a11d54ad00e40cc8d7fbee90db8b0837/source/containers.tex#L8773)
Effects: Constructs an empty vector, using the specified allocator[.](https://eel.is/c++draft/vector#cons-1.sentence-1)
[2](https://eel.is/c++draft/vector#cons-2)
[#](http://github.com/Eelis/draft/tree/7faaaf22a11d54ad00e40cc8d7fbee90db8b0837/source/containers.tex#L8778)
Complexity: Constant[.](https://eel.is/c++draft/vector#cons-2.sentence-1)
```